packages/cli/src/commands/j1Export.ts

```diff
@@ -45,6 +46,10 @@ export function j1Export() {
       '--no-include-deleted',
       'Exclude deleted entities/relationships in export',
     )
+    .option(
+      '--api-base-url <url>',
+      'The base URL used to initiate api calls with your instance of JupiterOne',
```

Comment:
Would you mind updating this with an example default?

```suggestion
      'The base URL used to initiate api calls with your instance of JupiterOne (defaults to `https://api.us.jupiterone.io`)',
```

Reply:
Updated both import and export with above suggestion.  Thanks!